.github/workflows/clippy.yaml

```
@@ -0,0 +1,20 @@
+name: run clippy
+
```

```suggestion
name: run clippy

permissions:
  pull-requests: write

```

CI のログを見たらコメントを出力できずサイレント爆死していそうだったので permission を渡してあげるといいかもしれない（検証はしてない、、）

https://github.com/orgs/community/discussions/26644
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

---

うーんこれだとだめだなぁ。。。

---

一旦マージして様子見します。なんか `forked repo` だと駄目だとか言ってるので、`master` に行けば治るのかもしれん。